test-project/bslint.json

```diff
@@ -1,17 +1,26 @@
 {
   "rules": {
-    "inline-if-style": "no-then",
-    "block-if-style": "no-then",
-    "condition-style": "no-group",
-    "named-function-style": "no-sub",
-    "anon-function-style": "no-sub",
-    "type-annotations": "off",
-    "assign-all-paths": "off",
-    "unsafe-path-loop": "error",
-    "unsafe-iterators": "error",
-    "unreachable-code": "warn",
-    "case-sensitivity": "warn",
-    "unused-variable": "warn",
-    "consistent-return": "error"
-  }
-}
+      "inline-if-style": "then",
```

Comment:
Can we use the vanilla `bslint` settings?

Reply:
Chris and I had gone through and picked all of the settings we prefer.

Which settings from our selections don't you prefer?

Reply:
I’m not a big fan of “function blah() as void”

but I can live with whatever you have here.